site/package.json

```diff
@@ -31,6 +31,7 @@
         "@babel/preset-react": "^7.18.6",
         "@babel/preset-typescript": "^7.21.0",
         "@testing-library/react": "^14.0.0",
+        "@types/lodash": "^4.14.191",
```

Comment:
We should have lodash itself as well in dependencies

Reply:
done